docs/coding-guidelines/javascript.md

```diff
@@ -68,50 +71,45 @@ foo( {
 } );
 
 foo( data, function() {
-    // Do stuff
+	// Do stuff
 } );
 
 foo( function() {
-    // Do stuff
+	// Do stuff
 }.bind( this ) );
 
 foo( function() {
-    // Do stuff
+	// Do stuff
 }, options );
-
 ```
 
 ## Examples of Good Spacing
 
 ```js
-var i;
-
 if ( condition ) {
-    doSomething( 'with a string' );
+	doSomething( 'with a string' );
 } else if ( otherCondition ) {
-    otherThing( {
-        key: value,
-        otherKey: otherValue
-    } );
+	otherThing( {
+		key: value,
+		otherKey: otherValue
+	} );
 } else {
-    somethingElse( true );
+	somethingElse( true );
 }
 
-// Unlike jQuery, WordPress prefers a space after the ! negation operator.
-// This is also done to conform to our PHP standards.
 while ( ! condition ) {
-    iterating++;
+	iterating++;
 }
 
-for ( i = 0; i < 100; i++ ) {
-    object[ array[ i ] ] = someFn( i );
-    $( '.container' ).val( array[ i ] );
+for ( let i = 0; i < 100; i++ ) {
+	object[ array[ i ] ] = someFn( i );
+	$( '.container' ).val( array[ i ] );
```

Comment:
yeah I got the point of it, but was just a bit irked that we used such a nasty example. we could have used a much simpler loop body